### DISCUSSION TITLE
How to switch to Traditional Chinese in Pinyin

### DISCUSSION THREAD

**Discussion by modestderp** [fcitx5-android/fcitx5-android#394] [Q&A | 问答] [answered]
**描述你请求的功能是否和问题有关 / Is your feature request related to a problem? Please describe.**
Currently when using Pinyin, only Simplified Chinese characters are displayed. It would be great if there was a switch to enable the user to type Traditional Chinese characters while using Pinyin as well.

**描述解决方案 / Describe the solution you'd like**
A toggle in settings when using Pinyin that allows the user to select between Simplified or Traditional Chinese as output.

**描述替代方案 / Describe alternatives you've considered**
I don't know the other Chinese inputs (Zhuying, Cangjie...etc) so I am unable to use alternatives when using fcitx5-android.

**附加信息 / Additional context**
There is a toggle in Trime that allows this switch, maybe could use that as reference?

**Comment by rocka** [comment, marked as answer]
Switch to traditional chinese in pinyin:

| 1 | 2 | 3 |
|:-:|:-:|:-:|
|![image](https://github.com/fcitx5-android/fcitx5-android/assets/13914967/2b515c8c-22fc-4225-9840-76f00b32579d)|![image](https://github.com/fcitx5-android/fcitx5-android/assets/13914967/7d107e1c-73b3-4c83-9ca0-ada3006b1956)|![image](https://github.com/fcitx5-android/fcitx5-android/assets/13914967/6438a7aa-5d1c-4e26-8ae1-14c7b3b420c3)|
